modlog_wiki_publisher.py

```
@@ -3,900 +3,1331 @@
+                target_author = action.target_author.name
+            else:
+                target_author = str(action.target_author)
+        
+        cursor.execute("""
+            INSERT OR REPLACE INTO processed_actions 
+            (action_id, action_type, moderator, target_id, target_type, 
+             display_id, target_permalink, removal_reason, target_author, created_at, subreddit) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+        """, (
+            action.id,
+            action.action,
+            get_moderator_name(action, False),  # Store actual name in database
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)